packages-internal/babel-plugin-resolve-imports/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@mui/internal-babel-plugin-resolve-imports",
+  "version": "1.0.16",
```

Comment:
That's unusual initial version ;)

Reply:
😄 I copied the `package.json` of `@mui/internal-scripts` and didn't give it any further thought.